packages/core/package.json

```diff
@@ -113,6 +113,7 @@
     "@opensearch-project/opensearch": "^2.3.1",
     "asn1js": "^3.0.5",
     "aws-sdk": "^2.1243.0",
+    "chart.js": "^3.9.1",
```

Comment:
Since we're not importing it in our code, but only in the browser, can we remove this?